docs/01-Configuration.md

```diff
@@ -0,0 +1,322 @@
+
+```
+[cosmos]
+key_derivation_path = "m/44'/118'/0'/0/0"
+```
+
+#### `msg_batch_size`
+
+Type: integer
+
+Used by the Orchestrator. The max number of Msgs to send to Sommelier at once.
+
+```
+[cosmos]
+msg_batch_size = 5
+```
+
+#### `prefix`
+
+Type: string
+
+The bech32 prefix to use for representing account addresses
+
+```
+[cosmos]
+prefix = "somm"
+```
+
+### `[cosmos.gas_price]` table
+
+For setting transaction fees
+
+#### `amount`
+
+Type: float
+
+The amount of `denom` you are willing to pay per gas for transactions.
+
+```
+[cosmos.gas_price]
+amount = 0.0
+```
+
+#### `denom`
+
+Type: string
+
+The denomination of the gas fee `amount`. Sommelier handles fees in `usomm`.
+
+```
+[cosmos.gas_price]
+denom = "usomm"
+```
+
+### `[ethereum]` table
+
+Config related to interaction with Ethereum
+
+#### `blocks_to_search`
+
+Type: integer
+
+How many Ethereum blocks back the Orchestrator should search for the last event nonce relayed to Cosmos.
+
+```
+[ethereum]
+blocks_to_search = 5000
+```
+
+#### `gas_price_multiplier`
+
+Type: float
+
+Multiplied by estimated gas fee per transaction. If your Ethereum transactions are failing due to insufficient gas, try increasing this value.
+
+```
+[ethereum]
+gas_price_multiplier = 1.0
+```
+
+#### `key_derivation_path`
+
+Type: string
+
+The path used to derive Etherum addresses from your private keys.
+
+```
+[ethereum]
+key_derivation_path = "m/44'/60'/0'/0/0"
+```
+
+#### `rpc`
+
+Type: string
+
+The node endpoint to be used for Ethereum interactions
+
+```
+[ethereum]
+rpc = "http://localhost:8545"
+```
+
+### `[gravity]` table
+
+Configuration related to the gravity bridge
+
+#### `contract`
+
+Type: string
+
+The address of the gravity contract on Ethereum
+
+```
+[gravity]
+contract = "0x0000000000000000000000000000000000000000"
+```
+
+#### `fees_denom`
+
+Type: string
+
+The denomination in which fees will be paid to relayers. This value must be a denom supported by the configured chain (whatever chain the node at `cosmos.grpc` is validating on).
+
+```
+[gravity]
+fees_denom = "usomm"
+```
+
+### `[keys]` table
+
+Keys config
+
+#### `delegate_key`
+
+Type: string
+
+The name of the key in `keystore` used for delegate signing by both Steward and the Orchestrator. The key name is the name of the key file *without the file extension*.
+
+```
+[keys]
+delegate_key = ""
+```
+
+### [metrics] table
+
+Config related to the Orchestrator metrics server
+
+#### `listen_addr`
+
+Type: string
+
+The server endpoint for monitoring Orchestrator metrics
+
+```
+[metrics]
+listen_addr = "127.0.0.1:3000"
+```
+
+### [server] table
+
+Config related to the Steward server
+
+#### `address`
+
+Type: string
+
+The IP address at which the Steward server will run
+
+```
+[server]
+address = "0.0.0.0"
+```
+
+#### `client_ca_cert_path`
+
+Type: string
+
+The path to the trusted CA used to sign the Strategy Provider's client certificate
```

Comment:
If fields are left out of the config, they are set to the default, correct? No one should be setting the client CA, we want to default to the current hardcoded PeggyJV CA. If there are other defaults we don't want people messing with, we should probably ask them not to set them.